src/widgets/PageButton.js

```diff
@@ -21,9 +21,10 @@ export function PageButton(props) {
   if (isDisabled) defaultTextStyle.push(globalStyles.disabledButtonText);
   return (
     <Button
-      {...buttonProps}
+      isDisabled={isDisabled}
```

Comment:
hmm, does this mean disabled buttons where not actually disabled

Reply:
I believe so